vector/src/nightly/java/im/vector/app/config/AnalyticsConfig.kt

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
It's a pain I had to copy paste all those files from `/release`. Still looking for a better solution.
But there are diff in this file and in file `shortcuts.xm` :/